languages/javascript/exercises/concept/classes-intro/.docs/introduction.md

```diff
@@ -0,0 +1,63 @@
+In general object-oriented programming, a class is an extensible template for storing associated values and methods.
+
+In JavaScript, a class is a special type of function that can bind values and methods to itself using the `this` keyword.
+
+Initialize classes by using the `class` keyword:
+
+```js
+
+class MyClass() {
+    //...
+}
+
+```
+
+To add a method (a function of a class) to a class, just write the function definition without the `function` keyword in the class body:
+
+```js
+class FunctionDemo {
```

Comment:
I would see this easier to understand without using the word 'function' in the class name and the methods.